File: launchdarkly/python/src/GetContextInstanceSegmentsMembershipByEnvExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.SegmentsApi(api_client).get_context_instance_segments_membership_by_env(
            project_key=None,
            environment_key=None,
            request_body=json.loads("""
                {
                    "address": {
                        "city": "Springfield",
                        "street": "123 Main Street"
                    },
                    "jobFunction": "doctor",
                    "key": "context-key-123abc",
                    "kind": "user",
                    "name": "Sandy"
                }
            """),
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling SegmentsApi#get_context_instance_segments_membership_by_env: %s\n" % e)

File: launchdarkly/php/src/GetContextInstanceSegmentsMembershipByEnvExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\SegmentsApi(config: $config))->getContextInstanceSegmentsMembershipByEnv(
        project_key: null,
        environment_key: null,
        request_body: json_decode(<<<'EOD'
            {
                "address": {
                    "city": "Springfield",
                    "street": "123 Main Street"
                },
                "jobFunction": "doctor",
                "key": "context-key-123abc",
                "kind": "user",
                "name": "Sandy"
            }
        EOD, true),
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling SegmentsApi#getContextInstanceSegmentsMembershipByEnv: {$e->getMessage()}";
}
